compiler/x/codegen/BinaryEvaluator.cpp

```diff
@@ -971,6 +971,20 @@ TR::Register *OMR::X86::TreeEvaluator::baddEvaluator(TR::Node *node, TR::CodeGen
    // target register generated and we return NULL to the caller (which should be
    // a store) to indicate that the store has already been done.
    //
+
+   if (NEED_CC(node))
+      {
+      TR_ASSERT(node->getOpCodeValue() == TR::badd,
```

Comment:
The assertion message should complain about the opcode and the node - not just the node